dbt_project.yml

```diff
@@ -33,10 +33,7 @@ models:
         +materialized: table
     staging:
       graph:
-        stg_nodes:
-          +materialized: table
-        stg_node_relationships:
-          +materialized: table
+        +materialized: table
```

Comment:
This change seems to be breaking Redshift. Is there a reason why we would want all models under staging/graph to materialized as tables?

Reply:
These all are logically identical entities that represent different sub-types of a node so for me it totally make sense to have them all as tables, but if this change is causing issues you can revert it. Definitely out of scope of this bug